vulcanai2/models/basenetwork.py

```diff
@@ -563,6 +566,9 @@ def fit(self, train_loader, val_loader, epochs,
                 fig_number = plt.gcf().number + 1 if plt.fignum_exists(1) else 1
                 plt.show()
 
+            if save_path:
+                time = get_time()
+                save_path = save_path + '/' + str(self.name) + '_' + time
```

Comment:
you should use the `get_save_path` no?

Reply:
@joemehltretter shouldn't it be something like `save_path = get_save_path(save_path, vis_type=self.name)` ? then we can get rid of `get_time()`

Reply:
@rfratila I update this in a recent push. I have it as `vis_type=str(self.name)`. I can remove the `str` if you'd like?

Reply:
yeah you don't need the `str()` since we check that it's string before when instantiating it.